### PR TITLE
Lesson 6 mosut sorin

### PR DESCRIPTION
**Pull request by sorinmosut**
add test file

### CODE REVIEW

File: lesson_6/mosut_sorin/test.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('Salut')
```
Comment (Liitle) [inline lesson_6/mosut_sorin/test.py:1]: Great print, but it is missing a new line at the end of file

Comment (VictorPop84) [reply lesson_6/mosut_sorin/test.py:1]: Beton COD

Comment (Munteanu-Alexandru) [reply lesson_6/mosut_sorin/test.py:1]: alo?aaaaloooo? sunt eu..picasoo

Comment (sorinmosut) [reply lesson_6/mosut_sorin/test.py:1]: i'll fix it

**Review by Liitle — APPROVED** [review]

**Review by Liitle — APPROVED** [review]

**Pull request merged**